pkg/acs/calacs/include/acs.h

```diff
@@ -13,6 +13,9 @@
 # define ACS_FITS_REC       82
 # define SZ_STRKWVAL        68
 
+#define ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN 4144
```

Comment:
Are you really specifically referring to ``ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN``? As this isn't actually used, it's really only for added understanding of what ``ACS_WFC_N_COLUMNS_PER_CHIP_EXCL_OVERSCAN`` is referring to.